src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractLanguageService`2.VsLanguageDebugInfo.cs

```diff
@@ -404,7 +404,7 @@ public int GetDataTipText(IVsTextBuffer pBuffer, VsTextSpan[] pSpan, out string
                             var subjectBuffer = editorAdapters.GetDataBuffer(pBuffer);
 
                             var textSnapshot = subjectBuffer.CurrentSnapshot;
-                            var document = textSnapshot.GetOpenDocumentInCurrentContextWithChanges();
+                            var document = CodeAnalysis.Text.Extensions.GetDocument(textSnapshot);
```

Comment:
(ditto)

Reply:
will manually fixup.  but this is some sort of rename issue.

Reply:
Missed this one?